src/app/zap-templates/zcl/tv-channel-cluster.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <domain>General</domain>
+    <name>TV Channel</name>
+    <code>0x0504</code>
+    <define>TV_CHANNEL_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>This cluster provides an interface for controlling the current TV Channel on a device.</description>
+    <attribute side="server" code="0x0000" define="TV_CHANNEL_LIST" type="OCTET_STRING" length="32" writable="false" optional="false">tv channel list</attribute>
+    <attribute side="server" code="0x0001" define="TV_CHANNEL_LINEUP" type="OCTET_STRING" length="32" writable="false" optional="false">tv channel lineup</attribute>
```

Comment:
is the non-list attribute also a limitation of current zap (I.e. I have to use octet_string instead of actual LineupInfo/ChannelInfo type)?

Reply:
I've fwd the question to the ZAP folks, I'm pretty sure it's unsupported today. 
Waiting for their confirmation.